docs/roadmap.markdown

```diff
@@ -4,9 +4,172 @@ title: "Roadmap"
 permalink: /roadmap/
 ---
 
-The Developer Toolkit is comprised of a set of tools that are worked on in
-collaboration with the community.  The following unified roadmap provides an
-overview of the current priorities across the toolkit, and links to the
-specific tool roadmaps.
+## Introduction
 
+This is a community driven project, sponsored by New Relic, to build a
+community around Observability as Code with New Relic. The following is the
```

Comment:
Try this instead:  "This documentation will define the initial problem space, identify who is impacted and provide a roadmap for success"